plugin-sysstat/lxqtsysstatcolours.cpp

```diff
@@ -58,30 +58,33 @@ LXQtSysStatColours::LXQtSysStatColours(QWidget *parent) :
     mDefaultColours[QStringLiteral("netReceived")]    = QColor("#000080");
     mDefaultColours[QStringLiteral("netTransmitted")] = QColor("#808000");
 
+    //
+    mShowColourMap[QString::fromLatin1("grid")] = ui->gridB;
+    mShowColourMap[QString::fromLatin1("title")] = ui->titleB;
+    mShowColourMap[QString::fromLatin1("cpuSystem")] = ui->cpuSystemB;
+    mShowColourMap[QString::fromLatin1("cpuUser")] = ui->cpuUserB;
+    mShowColourMap[QString::fromLatin1("cpuNice")] = ui->cpuNiceB;
+    mShowColourMap[QString::fromLatin1("cpuOther")] = ui->cpuOtherB;
+    mShowColourMap[QString::fromLatin1("cpuFrequency")] = ui->cpuFrequencyB;
+    mShowColourMap[QString::fromLatin1("memApps")] = ui->memAppsB;
+    mShowColourMap[QString::fromLatin1("memBuffers")] = ui->memBuffersB;
+    mShowColourMap[QString::fromLatin1("memCached")] = ui->memCachedB;
+    mShowColourMap[QString::fromLatin1("memSwap")] = ui->memSwapB;
+    mShowColourMap[QString::fromLatin1("netReceived")] = ui->netReceivedB;
+    mShowColourMap[QString::fromLatin1("netTransmitted")] = ui->netTransmittedB;
+
+    QMap<QString, QPushButton*>::const_iterator iterator = mShowColourMap.constBegin();
```

Comment:
I'd use `auto` instead of `QMap<QString, QPushButton*>::const_iterator`.

Reply:
ok